examples/IRB140/HandInput.m

```diff
@@ -0,0 +1,16 @@
+classdef HandInput < SingletonCoordinateFrame
+  % hand input coordinate frame
+  methods
+    function obj=HandInput(r, ind, name)
+      typecheck(r,'TimeSteppingRigidBodyManipulator');
```

Comment:
That seems reasonable to me. I'm all for avoiding duplicate code, and it would be great to use the same code for the hands regardless of what they're attached to.